android/src/main/java/com/swmansion/rnscreens/ScreenStackFragment.kt

```diff
@@ -118,16 +142,30 @@ class ScreenStackFragment : ScreenFragment, ScreenStackFragmentWrapper {
             // role. On top of that it breaks screens animations when alfa offscreen compositing is off
             // (which is the default)
             setBackgroundColor(Color.TRANSPARENT)
+
             layoutParams = AppBarLayout.LayoutParams(
-                AppBarLayout.LayoutParams.MATCH_PARENT, AppBarLayout.LayoutParams.WRAP_CONTENT
+                AppBarLayout.LayoutParams.MATCH_PARENT, getHeightOfToolbar(context)
             )
+
+            // On Material 3 the elevation is not visible on AppBarLayout.
+            // To prevent this behavior, we're setting outline shadow colors to black.
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+                outlineAmbientShadowColor = Color.BLACK
```

Comment:
Maybe it is something that could be controlled by the user?

Reply:
You mean the color of the elevation? I believe I've read somewhere the suggestion to add such modifier, so I believe we can do such thing! But let's maybe leave this idea for another PR.

Reply:
Great! Keep it in mind.

Reply:
@WoLewicki added the note as a feature on Screens agenda.